apps/rails/db/schema.rb

```
@@ -22,6 +21,29 @@
   create_enum "organizations_users_role", ["admin", "member"]
   create_enum "organizations_users_status", ["active", "inactive"]
 
+  create_table "agents", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "name"
+    t.string "retellai_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "calls", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "user_id"
```

RuboCop (1.69.1)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)